packages/neuron-ui/src/utils/const.ts

```diff
@@ -62,3 +62,34 @@ export enum Routes {
 export enum LocalStorage {
   Networks = 'networks',
 }
+
+export enum CapacityUnit {
+  CKB = 'ckb',
+  CKKB = 'ckkb',
+  CKGB = 'ckgb',
+}
+
+export const PlaceHolders = {
+  transfer: {
+    Address: 'eg: 0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674',
+    Capacity: 'eg: 100',
+  },
+}
+
+export const Tooltips = {
+  transfer: {
+    Address: 'Address to send capacity',
+    Capacity: 'Capacity to send',
```

Comment:
Maybe worthy a discussion, but it seems `Amount` is more easier to understand for end user than `Capacity`.